### PR TITLE
Make PlayerTags simpler and more generic.

### PR DESCRIPTION
**Pull request by kberg**
This relies on es2019 (Object.fromEntries) which is well supported on
browsers. There's another PR to move our lib to 2021, which I think is a
good move. But one thing at a time.

image

### CODE REVIEW

File: src/client/components/overview/PlayerTags.vue [hunk 1]
```diff
@@ -129,100 +97,103 @@ export default Vue.extend({
...
+      const vps = instance?.card.victoryPoints;
+      if (vps !== undefined && typeof(vps) !== 'number' && vps !== 'special' && vps.type !== 'resource') {
+        modifiers[vps.type].points += vps.points;
+        if (vps.per === 3) modifiers[vps.type].fraction = '⅓';
+      }
+    }
+
+    if (this.playerView.game.turmoil?.ruling === PartyName.UNITY &&
+      this.playerView.game.turmoil.politicalAgendas?.unity.policyId === 'up04') {
+      // space: discount += 2;
```
Comment (bafolts) [inline src/client/components/overview/PlayerTags.vue:135]: This conditional doesn't seem to do anything.

File: tsconfig.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
 {
     // Compiler options are mostly for the sub-configurations
     "compilerOptions": {
-        "lib": ["dom"],
+        "lib": ["dom", "es2021"],
```
Comment (bafolts) [inline tsconfig.json:6]: Is this intentionally changing to `es2021` from `es2019`?

Comment (kberg) [reply tsconfig.json]: Sorry, removing.

File: src/client/components/overview/PlayerTags.vue (hunk 1 above)
Comment (kberg) [reply src/client/components/overview/PlayerTags.vue]: Thanks, good catch. Fixed.

**Comment by kberg** [comment]
Ready for another pass.

**Review by bafolts — APPROVED** [review]

**Pull request merged**